developers/academy/zero_to_mvp/101_hello_weaviate/15_overview_vectors.mdx

```diff
@@ -9,56 +9,57 @@ import ReactPlayer from 'react-player/lazy'
 <ReactPlayer url='https://youtu.be/iFUeV3aYynI' controls='true'/>
 <br/>
 
-We've covered that Weaviate is a vector database, and that a vector search is similarity-based. But what is a vector?
+We've covered that Weaviate is a vector database, and that a vector search is similarity-based, but what is a vector?
```

Comment:
I personally start sentences with conjunctions more than most, as I think it can be a useful tool. In this case, I want to emphasise considering what a vector is. So I would prefer to leave it. 

I also know not everybody feels this way 😜 Something for the style guide perhaps.